src/sass/layout/widgets/widgets.scss

```
@@ -47,8 +47,13 @@ $padding-left-right: 30px;
   }
 
   &__item {
+    margin-bottom: 2.6rem;
     padding-right: $padding-left-right;
     padding-left: $padding-left-right;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
```

лучше тогда вот так

```suggestion
    &:not(:last-child) {
      margin-bottom: 2.6rem;
    }
```
а на 50-ой строке убрать

---

Сделано.